stan/math/prim/prob/neg_binomial_lpmf.hpp

```diff
@@ -102,19 +108,30 @@ return_type_t<T_shape, T_inv_scale> neg_binomial_lpmf(const T_n& n,
   }
 
   for (size_t i = 0; i < max_size_seq_view; i++) {
-    if (alpha_vec[i] > 1e10) {  // reduces numerically to Poisson
+    if (alpha_vec[i] > internal::neg_binomial_alpha_cutoff) {
+      // reduces numerically to Poisson
+      // The derivatives are obtained via Taylor series at alpha -> Inf
+      // via Mathematica as:
+      // nb[n_,alpha_,beta_]:= LogGamma[n + alpha] - LogGamma[n + 1] -
```

Comment:
I don't think we should assume the readers of comments understand mathematica.  So this should be put in Stan notation or just eliminated.  I couldn't understand what `LogGamma` is or `Log` or `D`.  If these comments stay, the functions used need to be explained.